scripts/temperature2

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Depends on lm-sensors (https://packages.ubuntu.com/bionic/lm-sensors)
+
+VALUE_FONT=${font:-$(xrescat i3xrocks.value.font "Source Code Pro Medium 13")}
+
+TEMP=$(sensors -u | grep -Po 'temp1_input: \K\d*')
+
+if [ "$TEMP" -gt 90 ]
```

Comment:
The conditional expression isn't parsing for me.  Testing:

```
$ ./temperature2
./temperature2: line 9: [: 37
27: integer expression expected
./temperature2: line 12: [: 37
27: integer expression expected
<span font_desc="Source Code Pro Medium 13" color="#D8DEE9">37
27℃</span>
```

My version of bash: 
```
$ bash --version
bash --version
GNU bash, version 5.0.3(1)-release (x86_64-pc-linux-gnu)
```

Reply:
I managed to reproduce and fix it. 
I also forced the command to only look at the first temperature it finds. This follows the logic of the existing [temperature](https://github.com/regolith-linux/regolith-i3xrocks-config/blob/master/scripts/temperature#L48-L53) script.